Kernel/Graphics/FramebufferDevice.h

```diff
@@ -59,6 +59,8 @@ class FramebufferDevice : public BlockDevice {
     OwnPtr<Region> m_real_framebuffer_region;
     OwnPtr<Region> m_swapped_framebuffer_region;
 
+    bool m_enabled { true };
```

Comment:
This name seems confusingly generic. Can we call it something like `m_graphics_mode_enabled` or if you can think of a good descriptive name for what it means to be "enabled" here :)